src/olympia/bandwagon/models.py

```diff
@@ -391,7 +391,8 @@ def post_delete(sender, instance, **kwargs):
             # present in the collection at the time, we also need to make sure
             # to invalidate add-ons that have been removed.
             Collection.update_featured_status(
-                sender, instance.collection, addons=[instance.addon], **kwargs)
+                sender, instance.collection,
+                addons=[instance.addon.pk], **kwargs)
```

Comment:
The problem was that we were passing add-on instances, which can be serialized so it doesn't work...